### PR TITLE
Micronaut cloud database guide base and Oracle Cloud

### PR DESCRIPTION
**Pull request by burtbeckwith**

### CODE REVIEW

**Comment by burtbeckwith** [comment]
I left some TODOs that I'll get back to in a bit and there's still some rework needed

**Comment by burtbeckwith** [comment]
Here's a PDF of the guide: [micronaut-cloud-database-oracle.pdf](https://github.com/micronaut-projects/micronaut-guides/files/8003990/micronaut-cloud-database-oracle.pdf)

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc]: I think this paragraph should be in a common snippet.

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc]: Pending

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc]: Pending

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc:549]: I don't understand the benefit of using this instead of a common snippet. Why did we needed `external`. Could not we just have?

```
common:micronaut-cloud-database-end.adoc

```

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc:432]: I don't understand the benefit of using this instead of a common snippet. Why did we needed `external`. Could not we just have?

```
common:micronaut-cloud-database-create-app.adoc[]
```

File: guides/micronaut-cloud-database-base/create-app.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-base/create-app.adoc:1]: Could not we just have this in a common snippet name `micronaut-cloud-database-create-app`?

File: guides/micronaut-cloud-database-base/end.adoc
Comment (sdelamo) [inline guides/micronaut-cloud-database-base/end.adoc]: Could not we just have this in a common snippet name `micronaut-cloud-database-end`?

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [reply guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc:298]: https://github.com/micronaut-projects/micronaut-guides/pull/743

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [reply guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc:549]: Resolved via https://github.com/micronaut-projects/micronaut-guides/pull/743

File: guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc
Comment (sdelamo) [reply guides/micronaut-cloud-database-oracle/micronaut-cloud-database-oracle.adoc:432]: Resolved via https://github.com/micronaut-projects/micronaut-guides/pull/743

**Comment by sdelamo** [comment]
I've added the PR to the project. Please, let's use the Github project.

File: buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy
Comment (sdelamo) [inline buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy]: Can you please add this change to a separate PR. So that we can merge it as soon as possible?

**Review by sdelamo — APPROVED** [review]

**Comment by sdelamo** [comment]
**WARNING: We cannot merge this until this PR is merged https://github.com/micronaut-projects/micronaut-guides/pull/685**

**Comment by sdelamo** [comment]
@kroegerk can you proofread this guide? Thanks!

File: buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy
Comment (burtbeckwith) [reply buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy]: https://github.com/micronaut-projects/micronaut-guides/pull/767

**Review by graemerocher — APPROVED** [review]

**Review by shingjo — APPROVED** [review]

File: guides/micronaut-cloud-database-oracle/metadata.json
Comment (graemerocher) [inline guides/micronaut-cloud-database-oracle/metadata.json]: Perhaps rename to `"Deploy a Micronaut MySQL database application to Oracle Cloud"` to clarify this is MySQL service

File: guides/micronaut-cloud-database-base/end.adoc
Comment (kroegerk) [inline guides/micronaut-cloud-database-base/end.adoc]:
```suggestion
Read more about https://micronaut-projects.github.io/micronaut-data/latest/guide/[Micronaut Data] and https://micronaut-projects.github.io/micronaut-oracle-cloud/latest/guide/[Micronaut Oracle Cloud Support].
```

Title case

File: guides/micronaut-cloud-database-oracle/metadata.json
Comment (kroegerk) [inline guides/micronaut-cloud-database-oracle/metadata.json]:
```suggestion
  "title": "Deploy a Micronaut Database Application to Oracle Cloud",
```

Title case

**Pull request merged**
